types/src/front/lib/connectors_api.ts

```diff
@@ -364,37 +364,6 @@ export class ConnectorsAPI {
     return this._resultFromResponse(res);
   }
 
-  // @deprecated
-  async getResourcesParents({
-    connectorId,
-    resourceInternalIds,
-  }: {
-    connectorId: string;
-    resourceInternalIds: string[];
-  }): Promise<
-    ConnectorsAPIResponse<{
-      resources: {
-        internalId: string;
-        parents: string[];
-      }[];
-    }>
-  > {
-    const res = await fetch(
-      `${CONNECTORS_API}/connectors/${encodeURIComponent(
-        connectorId
-      )}/resources/parents`,
-      {
-        method: "POST",
-        headers: this.getDefaultHeaders(),
-        body: JSON.stringify({
-          resourceInternalIds,
-        }),
-      }
-    );
-
-    return this._resultFromResponse(res);
-  }
-
```

Comment:
Can we get rid of get titles as well?

Reply:
Not yet -> this one needs to be merged first https://github.com/dust-tt/dust/pull/4065
(but I will get rid of it afterwards, it's on my todo)